.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: CI
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
Any reason not to cover the remaining Python impls? pycargoebuild has the full set.

Reply:
I only used the implementations that were already tested against in `tox.ini`. Do you want me to [drop <3.8 and add 3.11](https://github.com/projg2/eclean-kernel/pull/33#discussion_r1028715919)? And maybe add pypy?

Reply:
Yes, please. Update `tox.ini` too. Ideally let's do that + Actions separately from efistub, so I can merge it first and then use actions on this PR.

Reply:
https://github.com/projg2/eclean-kernel/pull/34